python/orca/src/bigdl/orca/learn/tf2/spark_runner.py

```diff
@@ -344,9 +347,11 @@ def step(self, data_creator, epochs=1, batch_size=32, verbose=1,
         else:
             stats = {k: v[-1] for k, v in history.history.items()}
         if self.rank == 0:
-            # if model_dir is not None:
-            #     model.save_weights(model_dir)
-            return ([weights, stats])
+            if self.model_dir is not None:
```

Comment:
how about just using `torch.save(model.state_dict(), PATH)`

Reply:
torch.save can support hdfs and s3?

Reply:
I meaning inside `save_pkl` and `load_pkl`?

Reply:
Sorry. This is TF. Orz